2/greeter.rb

```diff
@@ -1,2 +1,7 @@
-puts 'Hello, World!' do
+print 'Please enter your name:' do
```

Comment:
What are the `do` and `end` for here?

Reply:
I had originally used 'do' as this allowed the programme to run, and so I thought that this was the solution. I can see it is not!